manifests/09_deployment.yaml

```diff
@@ -43,7 +43,6 @@ spec:
         command: ["cluster-openshift-controller-manager-operator", "operator"]
         args:
         - "--config=/var/run/configmaps/config/config.yaml"
-        - "-v=4"
```

Comment:
It should not be important what is set here, but on a closer look it is! It seems OCM-o does not support obtaining verbosity through `OpenShiftControllerManager.operatorLogLevel` object. Seems like we have `-v=4` as a hack to have at least middle ground of logging here (`DEBUG`), so we can proceed with removal of this line.

OAS-o uses OperatorLoggingController here https://github.com/openshift/cluster-openshift-apiserver-operator/blob/eaaa4e6ce3e9ff1d2c4594406741ac10e98cb219/pkg/operator/starter.go#L290. We should do something similar in KAS-o before we remove this line.

Reply:
i will get a closer look too

Reply:
Yes OCM pods restart and seem to react to `logLevel` correctly. But changing the `operatorLogLevel` does not lead to change in OCM-o pod behaviour or a restart in my observation.

Reply:
> Yes OCM pods restart and seem to react to `logLevel` correctly. But changing the `operatorLogLevel` does not lead to change in OCM-o pod behaviour or a restart in my observation.

I think this is ok, change the operatorLogLevel will only restart the OCM pods, but not the operator itself

Reply:
yes no restart is correct and expected, nevertheless the log level change should occur in OCM-o pod

Reply:
we first have to fix the logging feature of the operator before we can remove this line

Reply:
> we first have to fix the logging feature of the operator before we can remove this line

I have commit a PR to fix the logging feature of the operator PR 292[](https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/292)

Reply:
with the https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/292 PR has merged